src/main/java/net/dv8tion/jda/core/requests/ratelimit/ClientRateLimiter.java

```diff
@@ -55,8 +52,6 @@ public Long getRateLimit(Route.CompiledRoute route)
     @Override
     protected void queueRequest(Request request)
     {
-        if (isShutdown)
-            throw new RejectedExecutionException("Cannot queue a request after shutdown");
```

Comment:
I was able to apply everything except for this, @Almighty-Alpaca should comment his intentions and apply changes as necessary.